README.md

```diff
@@ -279,38 +279,10 @@ The unit_test makefile target is used for running the unit tests:
 
 Please have a look to the section [on building the source code](#from-sources) in order to get more information about how to prepare the environment to run the unit_test target.
 
-## Advanced topics:
-
-* Installation and administration
```

Comment:
Why these links has been removed? I would have understand moving them to the new user/index.md and admin/index.md, but not removing thet completely. 

Note that although readthedocs.org would be a good documentation tool, I doesn't replace Markdown doc, which should bue self-contained. Modifying the .md files to improve the readthedocs.org documentation musn't ruin Markdown documentation iftself.

Or maybe I'm missing something that needs offline clarification... :)